.ci/jenkins/jenkins-trigger.groovy

```diff
@@ -0,0 +1,44 @@
+// This script acts as a trigger script for the main ngraph-unittest.groovy
+// Jenkins job.  This script is part of a Jenkins multi-branch pipeline job
+// which can trigger GitHub jobs more effectively than the GitHub Pull
+// Request Builder (GHPRB) plugin, in our environment.
+
+// The original ngraph-unittest job required the following parameters.  We
+// set these up below as global variables, so we do not need to rewrite the
+// original script -- we only need to provide this new trigger hook.
+//
+// Parameters which ngraph-unittest uses:
+String  PR_URL = CHANGE_URL
+String  PR_COMMIT_AUTHOR = CHANGE_AUTHOR
+String  JENKINS_BRANCH = "chrisl/new-ci-trigger"
+Integer TIMEOUTTIME = "3600"
+// BRANCH parameter is no loner needed
+// TRIGGER_URL parameter is no longer needed
+
+// Constants
+JENKINS_DIR="."
+
+env.MB_PIPELINE_CHECKOUT = true
+
+node("bdw && nogpu") {
```

Comment:
For your node labels, I'd suggest shifting to `trigger` so that we can load this top-level/trigger executor slot onto commodity hardware (that may or may not be a bdw). We can label a few nodes in Jenkins to properly hold this

Reply:
Change made